src/components/views/game/game/yourhand/YourHand.tsx

```diff
@@ -8,13 +15,53 @@ export const YourHand = () => {
   const { updatecards } = useCards()
   const { activeMatch, yourActiveHand } = useCurrentGame()
   const [opacity, setOpacity] = useState(10)
+  const [open, setOpen] = useState(false)
 
   const notYetPlayed =
     yourActiveHand?.cards.filter((value) => value.round === null) ?? []
 
+  //Check if all players did the score announcement for this match
   const clickCard = async (card, number) => {
     setOpacity(number)
-    updatecards(card)
+    if (activeMatch.matchState === Match.matchState.PLAYING) {
+      updatecards(card)
+    } else {
+      console.log("hallo")
+      setOpen(true)
+    }
+  }
+
+  // Close dialog
+  const handleClose = () => {
+    setOpen(false)
+  }
+
+  let content = <></>
+
+  // Inform the player that it is not possible to play a card
+  if (open) {
+    content = (
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">
+          {"Screw-your-neighbor: talk to other users"}
```

Comment:
```suggestion
          Screw-your-neighbor: talk to other users
```
denke ich geht auch ohne wrapping in typescript teil

Reply:
Lösche ich